polls/templates/polls/admin-management.html

```diff
@@ -23,14 +23,14 @@
             <!-- Annotator column -->
             {% for annotator in annotators %}
```

Comment:
I think these can be managed in a single for loop instead of multiple ones.

Reply:
True, I just think they needs to be independent of each other in other to be future proof, since the app isn't expecting hundreds of thousands of users, it'll be fine.